cmd/image-inspector.go

```diff
@@ -28,6 +29,8 @@ func main() {
 
 	flag.Parse()
 
+	inspectorOptions.AuthToken = os.Getenv("INSPECTOR_AUTH_TOKEN")
```

Comment:
the reasoning for using ENV is that CLI arguments are visible in a number of places, where ENV is less transparent (since this is a shared secret)